### PR TITLE
internal/imports: script for generating import path redirect files

### PR DESCRIPTION
**Pull request by jba**
This script fills a directory with the html files needed to redirect
import paths to gocloud.dev/...

Later, we'll configure Travis to run the script and push the generated
files to the gh-pages branch, where they will be served by GitHub Pages.

### CODE REVIEW

File: internal/imports/makeimports.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+fi
+
+
+shopt -s nullglob  # glob patterns that don't match turn into the empty string, instead of themselves
+
+function files_exist() {  # assumes nullglob
+  [[ ${1:-""} != "" ]]
+}
+
+rm -rf $OUTDIR
```
Comment (zombiezen) [inline internal/imports/makeimports.sh]: Optional: Seems like checking for existence and bailing is a bit safer than removing. Consider changing this.

Comment (jba) [reply internal/imports/makeimports.sh]: Done.

**Comment by jba** [comment]
@vangent ping

**Comment by jba** [comment]
nvm, I see you're not assigned.

**Pull request merged**